server/src/graphql/mutations/loginWithGithub.ts

```diff
@@ -77,7 +79,8 @@ export const loginWithGithub: MutationResolvers['loginWithGithub'] = async (
         avatarUrl: githubUser.avatar_url,
         email: email.email,
         githubAccessToken: data.access_token,
-        // TODO save refresh token + refresh_token_expires_in?
+        refreshToken: data.refresh_token,
+        refreshTokenExpiresIn: data.refresh_token_expires_in.toString(),
```

Comment:
Is `refresh_token_expires_in` a date or just an Int? Because if we store it like this we don't know exactly when it will expire. The best would be to change the field to `refreshTokenExpiresAt` which would be a date when the token will expire.

Reply:
@pradel it's an Int. So better to store as an Int?

Reply:
Well the int is relative to the time when you store it. Eg: 10 because it will expire in 10 hours.
If you just store 10 there is no way to know when it will expire as the object could have been updated 5 hours or 1 month ago.
Storing the date when it will expire is the best as it can be understood at any time we do a user query in the future.